packages/rest-hooks/src/restHooksTypes.ts

```diff
@@ -176,130 +178,95 @@ export namespace StcRest {
     */
   export type CreateGetFn = (
     restPath: string,
-      // Path to the REST endpoint (including query parameters if desired)
-      // Can contain path variables in the form of `:variableToSubstitute`.
+    // Path to the REST endpoint (including query parameters if desired).
+    // Can contain path variables in the form of `:variableToSubstitute`.
     restParams?: RestParams,
-      // Optional path and query parameters for the REST call.
+    // Optional path and query parameters for the REST call.
     axiosOptions?: Partial<AxiosRequestConfig>
-      // Optional Axios request configuration.
-    ) => GetFn;
-
-
-    export interface MutateFnOptions {
-      data: unknown;
-      restParams?: RestParams;
-    }
-
-
-  // TODO: convert the docs below from tsdoc to match the documentation style
-  // of the docs above.  Also, double check to make sure the docs are correct,
-  // as there has beens some churn
+    // Optional Axios request configuration.
+  ) => GetFn;
+
+  // Options passed into the MutateFn.
+  export interface MutateFnOptions {
+    data: unknown;
+    // The parameters for the mutation operation.
+    restParams?: RestParams;
+    // restParams.pathParams - For path variable substitutions.
+    // restParams.queryParams - Optional path and query params for the REST call.
+  }
 
- /**
-   * A function that performs a REST mutation (create or update) operation.
+  /**
+   * Function that performs a REST mutation (create or update) for specified path and rest params.
```

Comment:
I prefer not to have the stars "*" in the body of the header documentation, as to me they add clutter